### PR TITLE
Release v0.29.0

### PR DESCRIPTION
**Pull request by Farhad-Shabani**

### CODE REVIEW

File: .changelog/v0.29.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This release includes the latest Tendermint-rs v0.29.0 and replaces the
+old API with the new `ValidationContext`/`ExecutionContext` API as the default.
```
Comment (plafer) [inline .changelog/v0.29.0/summary.md]:
```suggestion
This release includes the latest Tendermint-rs v0.29.0 and removes the
`Reader` and `Keeper` API in favor of the new `ValidationContext`/`ExecutionContext` API as the default.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
 # CHANGELOG
 
+## v0.29.0
+
+*February 22, 2023*
+
+This release includes the latest Tendermint-rs v0.29.0 and replaces the
+old API with the new `ValidationContext`/`ExecutionContext` API as the default.
```
Comment (plafer) [inline CHANGELOG.md]:
```suggestion
This release includes the latest Tendermint-rs v0.29.0 and removes the
`Reader` and `Keeper` API in favor of the new `ValidationContext`/`ExecutionContext` API as the default.
```

File: crates/ibc/src/lib.rs [hunk 3]
```diff
@@ -13,7 +13,7 @@
 #![forbid(unsafe_code)]
 // https://github.com/cosmos/ibc-rs/issues/342
 #![allow(clippy::result_large_err)]
-#![doc(html_root_url = "https://docs.rs/ibc/0.28.0")]
+#![doc(html_root_url = "https://docs.rs/ibc/0.29.0")]
```
Comment (plafer) [inline crates/ibc/src/lib.rs]: let's remove this (see guideline https://github.com/rust-lang/api-guidelines/issues/229) and update CONTRIBUTING.md as well

Comment (Farhad-Shabani) [reply crates/ibc/src/lib.rs]: [f28f09d](https://github.com/cosmos/ibc-rs/pull/461/commits/f28f09d86fe41ad419cb69975d43bd36b38828f7)

**Review by plafer — APPROVED** [review]

**Pull request merged**